src/software/new_geom/util/physics.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "software/new_geom/point.h"
+
+/**
+ * Calculates the future position of a moving object
+ *
+ * @pre The units of the parameters are consistent, e.g. accleration is in (units of
+ * initial_position)/(units of time_in_future)^2
+ *
+ * @param initial_position The initial position of the object
+ * @param initial_velocity The initial velocity of the object in position
+ * @param acceleration The constant acceleration applied to the object over time
+ * @param time_in_future The time in the future to estimate position
+ *
+ * @return future position of the object
+ */
+Point calculateFuturePosition(const Point &initial_position,
```

Comment:
Could we rename to `calculateFuturePositionWithConstantAcceleration`? I know it's a bit wordy, but I think it's a _really_ important point to be explicit about the constant acceleration assumption we're making here. 

Similarly for `calculateFutureVelocity` below.

Reply:
I know there was a comment about allowing for polymorphism in a previous PR.

Either/or works for me, but something to keep on mind here. Having a constant acceleration as a parameter implies the prediction will use const-acc IMO

Reply:
I have renamed the param